harness/python/feature.py

```diff
@@ -75,13 +75,8 @@ async def run(self) -> None:
         self.client = await Client.connect(self.address, namespace=self.namespace)
 
         # Run worker
-        self.worker = Worker(
-            self.client,
-            task_queue=self.task_queue,
-            workflows=self.feature.workflows,
-            activities=self.feature.activities,
-        )
-        async with self.worker:
+        self.start_worker()
```

Comment:
You'll want to add `self.worker: Optional[Worker] = None` as the last statement in `__init__`.